certified-connectors/Narvar/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1795 @@
+                          },
+                          "item_image": {
+                            "type": "string",
+                            "description": "This must be a url to a public image",
+                            "title": "item_image"
+                          },
+                          "sku": {
+                            "type": "string",
+                            "description": "This value is used as the ID for the item object (Primary Key for@OrderItem objects)",
+                            "title": "sku"
+                          },
+                          "name": {
+                            "type": "string",
+                            "description": "Item name",
+                            "title": "name"
+                          },
+                          "quantity": {
+                            "type": "integer",
+                            "format": "int32",
+                            "description": "Item quantity",
+                            "title": "quantity"
+                          },
+                          "item_promise_date": {
+                            "type": "string",
+                            "description": "Shipment promise date.",
+                            "title": "item_promise_date"
+                          }
+                        }
+                      },
+                      "description": "order_items"
+                    },
+                    "shipments": {
+                      "type": "array",
+                      "items": {
+                        "type": "object",
+                        "properties": {
+                          "items_info": {
+                            "type": "array",
+                            "items": {
+                              "type": "object",
+                              "properties": {
+                                "quantity": {
+                                  "type": "integer",
+                                  "format": "int32",
+                                  "description": "quantity"
+                                },
+                                "sku": {
+                                  "type": "string",
+                                  "description": "sku"
+                                },
+                                "item_id": {
+                                  "type": "string",
+                                  "description": "item_id"
+                                }
+                              }
+                            },
+                            "description": "items_info"
+                          },
+                          "carrier": {
+                            "type": "string",
+                            "description": "Normalized carrier moniker associated to the given tracking number.This value must match one of the Narvar Carrier Enums. Please check with your Narvar project team if you are unsure what to use.",
```

Comment:
@BekaAre I would suggest you keep all parameters' descriptions short to fit in the text box. Please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#parameters).

Reply:
Updated accordingly.Please let me know if it is ok.